en/appendices/4-0-migration-guide.rst

```diff
@@ -107,6 +107,8 @@ Component
 * ``Cake\Controller\Component\PagingComponent`` now sets paging params info as
    request attribute instead of request param. Hence you should now use
   ``$request->getAttribute('paging')`` instead of ``$request->getParam('paging')``.
+* The ``requireSecure`` behavior of ``SecurityComponent`` was removed. Instead
```

Comment:
The feature isn't still removed from SecurityComponent. Wouldn't we be marking it as deprecated instead?

Reply:
Ooops. I'll update this note.